engines/director/debugger/dt-internal.h

```diff
@@ -106,16 +106,20 @@ typedef struct ImGuiState {
 		int _typeFilter = 0x7FFF;
 	} _cast;
 	struct {
-		Common::Array<ImGuiScript> _scripts;
-		uint _current = 0;
+		struct ScriptData {
+			Common::Array<ImGuiScript> _scripts;
+			uint _current = 0;
+			bool _showByteCode = false;
+			bool _showScript = false;
+		} *_scriptData;
```

Comment:
This design is not clear to me, as you just hold here pointer to `_windowScriptData`